python/examples/experimental/Multi dataset logger.ipynb

```diff
@@ -0,0 +1,118 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "d026913b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collecting whylabs-client\n",
+      "  Downloading whylabs_client-0.4.2-py3-none-any.whl (370 kB)\n",
+      "\u001b[2K     \u001b[38;2;114;156;31m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m370.7/370.7 kB\u001b[0m \u001b[31m6.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\n",
+      "\u001b[?25hRequirement already satisfied: python-dateutil in /home/anthony/workspace/whylogs/python/.venv/lib/python3.10/site-packages (from whylabs-client) (2.8.2)\n",
+      "Requirement already satisfied: urllib3>=1.25.3 in /home/anthony/workspace/whylogs/python/.venv/lib/python3.10/site-packages (from whylabs-client) (1.26.14)\n",
+      "Requirement already satisfied: six>=1.5 in /home/anthony/workspace/whylogs/python/.venv/lib/python3.10/site-packages (from python-dateutil->whylabs-client) (1.16.0)\n",
+      "Installing collected packages: whylabs-client\n",
+      "Successfully installed whylabs-client-0.4.2\n",
+      "Note: you may need to restart the kernel to use updated packages.\n"
+     ]
+    }
+   ],
+   "source": [
+    "%pip install whylabs-client\n",
+    "# Typically you would `pip install whylogs[whylabs]` for dependencies\n",
```

Comment:
This notebook will fail if you run it by itself because there is no whylogs installed, can you include something like:
%pip install 'whylogs[whylabs]'

Reply:
Actually a little confused about how this is "supposed" to work. If I do that then the notebook will fail right now because we haven't published yet. It actually works right now when run inside of poetry shell though. Is that still the right thing to do?